code/ZAS/_gas_mixture.dm

```diff
@@ -154,9 +154,8 @@
 	if(thermal_energy < 0)
 		if(temperature <= min_temp)
 			return 0
-		var/thermal_energy_limit = -(temperature - min_temp) * heat_capacity	//ensure temperature does not go below min_temp
-		thermal_energy = max(thermal_energy, thermal_energy_limit)	//thermal_energy and thermal_energy_limit are negative here.
-	temperature += thermal_energy/heat_capacity
+		thermal_energy = max(thermal_energy, (min_temp - temperature) * heat_capacity) //ensure temperature does not go below min_temp. thermal_energy is negative here.
```

Comment:
was this file meant to be included?

Reply:
Yes, I inadvertently removed my mentioning of making a few other minor changes when editing the PR body.